### PR TITLE
add -a flag to disable alias reversing

### PR DESCRIPTION
**Pull request by crclark96**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -7,11 +7,16 @@
 
 ginh is not a histogram
 
-usage: `./ginh.sh [-h] [-n entries] [-f hist_file] [-c chart_char] [-l line_len]`
+usage: `./ginh.sh [-h] [-a] [-n entries] [-f hist_file] [-c chart_char] [-l line_len]`
```
Comment (theryangeary) [inline readme.md:10]: Do you think it would be a good idea to add the explanation of these flags to the help/usage output?

File: ginh.sh [hunk 2]
```diff
@@ -175,7 +179,9 @@ fi
 
 filters+=("fish_filter")
 filters+=("zsh_extended_filter")
-filters+=("reverse_aliases_filter")
+if [ $alias -eq 1 ]; then
```
Comment (theryangeary) [inline ginh.sh]: would it be cleaner to instead only set alias to a value when the ginh's -a flag is used, and then here check if alias is set with bash's -z flag?

File: readme.md (hunk 1 above)
Comment (crclark96) [reply readme.md:10]: check lines 16-19

File: ginh.sh (hunk 2 above)
Comment (crclark96) [reply ginh.sh]: yep, adding

**Review by theryangeary — APPROVED** [review]
I made a new project item to update the documentation, so I think this looks good :+1: 
Can you rebase this so the git history looks pretty :face_with_head_bandage:

**Pull request merged**